portfolio/pom.xml

```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```

Comment:
I don't understand why the jetty version is lower.

Reply:
Yes haha this was due to an implementation error that was communicated to us by kevin! Thank you!